src/ui/linkformview.js

```diff
@@ -210,6 +248,84 @@ export default class LinkFormView extends View {
 
 		return button;
 	}
+
+	/**
+	 * Populates {@link module:ui/viewcollection~ViewCollection} of {@link module:ui/button/switchbuttonview~SwitchButtonView}
+	 * made based on {@link #manualDecorators}
+	 *
+	 * @private
+	 * @returns {module:ui/viewcollection~ViewCollection} of Switch Buttons.
+	 */
+	_createManualDecoratorsUIView() {
+		const switches = this.createCollection();
+
+		switches.bindTo( this.manualDecorators ).using( item => {
```

Comment:
Does the `this.manualDecorators` collection change over time? Are new decorators coming and some other leaving it? If not, why bind `switches` to it?

Reply:
no, those should be read from config only during editor initialization.